lua/lspconfig/server_configurations/postgres_lsp.lua

```diff
@@ -0,0 +1,20 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'postgres_lsp' },
+    filetypes = { 'sql' },
+    root_dir = util.root_pattern 'root-file.txt',
```

Comment:
This seems like a bad copy-paste

Reply:
Could you elaborate on that? I know it looks weird but it seems to be the way to do it https://github.com/supabase/postgres_lsp/blob/47dd0132b12661ab6c97f5fba892e567a5109c84/root-file.txt

Reply:
Wow. Please ignore my comment :). What a strange name for this file.